proto/buf/alpha/audit/v1alpha1/event.proto

```diff
@@ -80,11 +100,145 @@ message Event {
+    PayloadOrganizationMemberRemoved organization_member_removed = 10;
+    PayloadRepositoryCreated repository_created = 11;
+    PayloadRepositoryDeleted repository_deleted = 12;
+    PayloadRepositoryPushed repository_pushed = 13;
+    PayloadRepositoryMemberAdded repository_member_added = 14;
+    PayloadRepositoryMemberRoleChanged repository_member_role_changed = 15;
+    PayloadRepositoryMemberRemoved repository_member_removed = 16;
+    PayloadPluginCreated plugin_created = 17;
+    PayloadPluginDeleted plugin_deleted = 18;
+    PayloadPluginPushed plugin_pushed = 19;
+    PayloadUserRegistered user_registered = 20;
+    PayloadUserDeactivated user_deactivated = 21;
+    PayloadUserDeleted user_deleted = 22;
+    PayloadUserLoggedIn user_logged_in = 23;
+    PayloadUserLoggedOut user_logged_out = 24;
   }
 }
 
-// PayloadOrganizationCreated is the payload for an ORGANIZATION_CREATED event.
-message PayloadOrganizationCreated {
-  // created_time is when the organization was created.
-  google.protobuf.Timestamp created_time = 1;
+message PayloadOrganizationCreated {}
+
+message PayloadOrganizationDeleted {}
+
+message PayloadOrganizationMemberAdded {
+  // member_id is the id of the member added to the organization.
+  string member_id = 1;
+  // member_name is the name of the member added to the organization.
+  string member_name = 2;
+  // member_role is the role granted to the member added to the organization.
+  buf.alpha.registry.v1alpha1.OrganizationRole member_role = 3;
+}
+
+message PayloadOrganizationMemberRoleChanged {
+  // organization_id is the id of the organization within which the role was changed.
+  string organization_id = 1;
+  // organization_name is the name of the organization within which the role was changed.
+  string organization_name = 2;
+  // old_role is the old role of the member whose role was changed.
+  buf.alpha.registry.v1alpha1.OrganizationRole old_role = 3;
+  // new_role is the new role of the member whose role was changed.
+  buf.alpha.registry.v1alpha1.OrganizationRole new_role = 4;
+}
+
+message PayloadOrganizationMemberRemoved {
+  // member_id is the id of the member removed from the organization.
+  string member_id = 1;
+  // member_name is the name of the member removed from the organization.
+  string member_name = 2;
+  // member_role is the role that the member had when removed from the organization.
+  buf.alpha.registry.v1alpha1.OrganizationRole member_role = 3;
+}
+
+message PayloadRepositoryCreated {
+  // owner_id is the id of the owner of the repository.
+  string owner_id = 1;
+  // owner_name is the name of the owner of the repository.
+  string owner_name = 2;
 }
+
+message PayloadRepositoryDeleted {
+  // owner_id is the id of the owner of the repository.
+  string owner_id = 1;
+  // owner_name is the name of the owner of the repository.
+  string owner_name = 2;
+}
+
+message PayloadRepositoryPushed {
```

Comment:
Not blocking, just an open thought: the resource for this event is the new commit. It has an ID, but not sure if we expose that in anyway. Commits are always dealt with the name (hash). Food for thought, should we include the ID? or have an empty ID? or have the same value as ID and name?

Reply:
I guess I'd expect ID to be the "public" ID and name to be the hash? The both are exposed in protos, I believe: https://github.com/bufbuild/buf/blob/781efb570e17049fa88920f73f05dd008e760460/proto/buf/alpha/registry/v1alpha1/repository_commit.proto#L23-L32

Reply:
Nice! yes let's do that